Source/UltraleapTrackingCore/UltraleapTracking_APL.xml

```diff
@@ -95,20 +95,43 @@
 
 			public void onUnbound()
 			{
-				Log.debug("UltraleapTracking OnUnBound() called");
+			Log.debug("UltraleapTracking OnUnBound() called");
 			}
 			};
 			private ServiceBinder serviceBinder;
+
+
+			public void AndroidThunkJava_Bind()
+			{
+			if (ServiceBinder.isServiceInstalled(getApplicationContext()))
+			{
+			Log.debug("UltraleapTracking calling bind() on the service binder from LeapComponent.cpp");
+			serviceBinder.bind();
+			}
+			}
+
+			public void AndroidThunkJava_Unbind()
+			{
+			Log.debug("UltraleapTracking calling unbind() on the service binder from LeapComponent.cpp");
```

Comment:
Also, this code might need the same service binder installed check as above, just in case someone calls it without ever calling Bind?